index.html

```diff
@@ -168,69 +228,144 @@ <h2>Project Art Printing Data</h2>
+          standard</p>
+        <div class="lang">
+          <ul>
+            <li><button>html</button></li>
+            <li><button>bootstrap</button></li>
+            <li><button>Ruby</button></li>
+          </ul>
+        </div>
+       
+      </div>
 
 
+      <div class="grid item11 desktop">
+        <h2>Project Art Printing Data</h2>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's
+          standard</p>
+        <div class="lang">
+          <ul>
+            <li><button>html</button></li>
+            <li><button>bootstrap</button></li>
+            <li><button>Ruby</button></li>
+          </ul>
+        </div>
+        
+      </div>
 
-  <section class="Programming-skills">
-    <ul>
-      <li id="languages">
-        <h4 class="first">Languages<h4>
-            <ul>
-              <li>Javascript</li>
-              <li>Ruby</li>
-              <li>Html</li>
-              <li>Css</li>
-            </ul>
-      </li>
-      <li id="frameworks">
-        <h4>Frameworks<h4>
-            <ul>
-              <li>Bootstrap</li>
-              <li>Ruby on Rails</li>
-              <li>RsPec</li>
-              <li>CapyBara</li>
-              <li>Selenium</li>
-            </ul>
-      </li>
-      <li id="skillset">
-        <h4>Skills<h4>
-            <ul>
-              <li>Codekit</li>
-              <li>Github</li>
-              <li>Codepen</li>
-              <li>Gitlab</li>
-              <li>Terminal</li>
-            </ul>
-      </li>
 
-    </ul>
+      <div class="grid item12 desktop">
+        <h2>Data Dashboard Healthcare</h2>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's
+          standard</p>
+        <div class="lang">
+          <ul>
+            <li><button>html</button></li>
+            <li><button>bootstrap</button></li>
+            <li><button>Ruby</button></li>
+          </ul>
+        </div>
+        
+      </div>
+
+
+      <div class="grid item13 desktop">
+        <h2 class="web">Website Portfolio</h2>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's
+          standard</p>
+        <div class="lang">
+          <ul>
+            <li><button>html</button></li>
+            <li><button>bootstrap</button></li>
+            <li><button>Ruby</button></li>
+          </ul>
+        </div>
+        
+      </div>
+
 
+    </div>
   </section>
 
+
+  <div class="bottom-dark">
+    <section class="about">
+      <h3>ABOUT ME</h3>
+      <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work
+        and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <button class="see-project btn2">Get my Resume</button>
+
+    </section >
+    <hr class="foot">
```

Comment:
The about section looks amazing 👌 but don't you think the font size and spacing are not the same as the one in the Figma design? You can significantly improve your design by following these rules. Kindly make sure to update this.

Yours
![image](https://user-images.githubusercontent.com/34779928/206299087-bfaca988-12c1-401e-82f3-8ed07e71bb24.png)

Original
![image](https://user-images.githubusercontent.com/34779928/206299146-cc662b8b-033b-453e-8172-8073fbdfa716.png)